File: CRUD_VB/Form1.vb
Public Class Form1
    Private Sub btnGetClients_Click(sender As Object, e As EventArgs) Handles btnGetClients.Click
        Dim controlador As ClientController = New ClientController

        For Each item In controlador.GetClient().ToList()
            listBox1.Items.Add(item)
        Next
    End Sub

    Private Sub btnActualizar_Click(sender As Object, e As EventArgs) Handles btnActualizar.Click
        Dim controller As ClientController = New ClientController()
        Dim client As Client = New Client With {
            .ClientId = 1,
            .FullName = "Luis Contreras"
        }
        controller.UpdateClient(client)
    End Sub

    Private Sub btnEliminar_Click(sender As Object, e As EventArgs) Handles btnEliminar.Click
        Dim controller As ClientController = New ClientController()
        Dim model As Client = New Client With {
            .ClientId = 3
        }
        controller.DelClient(model)
    End Sub

    Private Sub btnAgregar_Click(sender As Object, e As EventArgs) Handles btnAgregar.Click
        Dim controlador As ClientController = New ClientController()
        Dim model As Client = New Client With {
            .FullName = "Prueba",
            .Email = "prueba@prueba.com",
            .PhoneNumber = "2781231"
        }
        controlador.AddClient(model)
    End Sub
End Class

File: CRUD_CSharp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CRUD_CSharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnGetClients_Click(object sender, EventArgs e)
        {
            ClientController controlador = new ClientController();
            foreach (var item in controlador.GetClient().ToList())
            {
                listBox1.Items.Add(item);
            }
        }

        private void btnAgregar_Click(object sender, EventArgs e)
        {
            ClientController controlador = new ClientController();
            Client model = new Client
            {
                FullName = "Prueba",
                Email = "prueba@prueba.com",
                PhoneNumber = "2781231"
            };
            controlador.AddClient(model);
        }

        private void btnEliminar_Click(object sender, EventArgs e)
        {
            ClientController controller = new ClientController();
            Client model = new Client
            {
                ClientId = 2
            };
            controller.DelClient(model);
        }

        private void btnActualizar_Click(object sender, EventArgs e)
        {
            ClientController controller = new ClientController();
            Client client = new Client
            {
                ClientId = 1,
                FullName = "Luis Contreras Arroyo"
            };
            controller.UpdateClient(client);
        }
    }
}
